legacy/tensormap-server/app/common/make_model_json.py

```diff
@@ -44,36 +51,35 @@ def makeKerasModel(json_config):
     hiddenNo = 0
     parentId = "userModel"
 
-    for i in range(len(json_config["graph"][0]["layers"])):        
-        print("i:",i) 
+    for i in range(len(json_config["graph"][0]["layers"])):
+        print("i:", i)
         print(parentId)
 
-        for k in range(len(json_config["graph"][0]["layers"])):  
-            print("k:",k)
-            print("parent k: ",json_config["graph"][0]["layers"][k]["parentNode"])
-            
+        for k in range(len(json_config["graph"][0]["layers"])):
+            print("k:", k)
+            print("parent k: ", json_config["graph"][0]["layers"][k]["parentNode"])
 
             if json_config["graph"][0]["layers"][k]["parentNode"] == parentId:
-                layer_dict = controlLogic(json_config,k)
+                layer_dict = controlLogic(json_config, k)
                 dict_list.append(layer_dict)
                 parentId = json_config["graph"][0]["layers"][k]["id"]
                 print(parentId)
-                break             
-
-    layerCommon = {"name": "userModel", "layers":dict_list}
-    modelCommon = {"class_name": "Sequential", "config":layerCommon, "keras_version": "2.2.4-tf", "backend": "tensorflow"}
-    #Only for testing
-    modelCommon["config"]["layers"][0]["input_shape"] =  [1000,]
+                break
+
+    layerCommon = {"name": "userModel", "layers": dict_list}
+    modelCommon = {
+        "class_name": "Sequential",
+        "config": layerCommon,
+        "keras_version": "2.2.4-tf",
+        "backend": "tensorflow",
+    }
```

Comment:
Make a constant parameter file and define these parameters there and invoke in here

Reply:
it's done

Reply:
it' Done